src/app/shared/context.service.ts

```diff
@@ -197,6 +197,10 @@ export class ContextService {
   }
 
   private checkForReservedWords(arg: string): boolean {
+    // All words starting with _ are reserved
+    if (arg.startsWith('_')) {
```

Comment:
Is the underscore only being added to try to identify reserved words in the url?  Why not just use the named route params instead so :entity or :space values can be extracted.